lib/ansible/modules/system/nfsexport.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2017 Randy Nance
+# based on code by Antonio Messina, Copyright (C) 2013, GC3, University of Zurich. All rights reserved.
+#
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+
+__docformat__ = 'reStructuredText'
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = """
+---
+module: nfsexport
+version_added: "2.4"
+short_description: This module will configure a nfs exports file
+description:
+    - This module will configure an nfs share by writing an entry in the exports file.
+notes:
+    - This module only changes the exports file itself. If changes are made, the
+      exportfs command should be run and the nfs daemon reloaded.
+author: "Randy Nance (randynobx) <randynobx@gmail.com>"
+options:
+    path:
+        description:
+            - Directory to export.
+        required: true
+    dest:
+        description:
+           - Export file to write to.
+        default: /etc/exports
+    clients:
+        description:
+            - list of nfs clients and host specific options.
+        default: "*"
+    options:
+        description:
+            - Nfs export options for all clients in export entry.
+    state:
+        description:
+            - Whether to add or remove export path from file.
+        default: present
+        choices:
+            - present
+            - absent
+"""
+
+EXAMPLES = """
+- name: ensure "/home 10.0.0.1(rw)" exists in /etc/exports
+  nfsexport:
+    path: /home
+    clients: 10.0.0.1(rw)
+    state: present
+
+- name: ensure "/nfs/share -sync,root_squash 10.0.0.1(rw) 10.1.0.0/24(ro)" exists in /etc/exports
+  nfsexport:
+    path: /nfs/share
+    clients:
+      - 10.0.0.1(rw)
+      - 10.1.0.0/24(ro)
+    options: sync,root_squash
+
+- name: remove "/home/user *" from /etc/nfsexports if present
+  nfsexport:
+    path: /home/user
+    dest: /etc/nfsexports
+    state: absent
+
+- name: ensure "/nfs/share *(ro,sync)" exists in /etc/exports
+  nfsexport:
+    path: /nfs/share
+    options: ro,sync
+"""
+
+RETURN = """
```

Comment:
We should fill the RETURN variable, but right now it should be `RETURN = ''' # '''`

> If your module doesn’t return anything (apart from the standard returns), you can use RETURN = ''' # '''.